doc/docs/master/concepts/pipeline-concepts/datum/relationship-between-datums.md

```diff
@@ -38,127 +29,84 @@ and data generation workflows.
 For more information about glob patterns, see
 [Glob Pattern](glob-pattern.md).
 
-Disregarding of how many datums you define and how many
+### 2- **Transformation**
+The pipeline **uses your code to process the datums**.
+
+!!! Note "Reminder"
+        The output produced by a pipeline's job is written to an output repo of the same name. However, you do not need to specify this name explicitly; you write your output to `/pfs/out`. 
+### 3- **Creation of output datums**    
```

Comment:
I'm not sure I understand this section. There isn't really a notion of input / output datums, just the datums for the job. The datums have input files associated with them, and the user code may output files after processing those input files. Also, just to be clear, the number of input files for a datum may not be the same as the number of files output by the user code.

Reply:
I removed the term `output datum` entirely since it does not relate to any existing concept in Pachyderm. Changed for `output files`.